workflow/schemas/config.schema.yaml

```
@@ -17,3 +17,20 @@ required:
   - units
   - resources
   - default_container
+
+padd_target_regions:
+    type: object
+    description: parameters for padd_target_regions
+    properties:
+      benchmark_repeats:
+        type: integer
+        description: set number of times benchmark should be repeated for padd_target_regions
```

```suggestion
        description: set number of times benchmark should be repeated
```

suggestion for simplification of the descriptions